.github/workflows/snyk-pr.yml

```diff
@@ -3,7 +3,7 @@ name: Snyk PR
 on:
   pull_request:
     paths:
-      - 'package.json'
+      - '**/package.json'
```

Comment:
** is a recursive wildcard expression, meaning any number of directories deep. 

I think it would only be child directories if expressed as ./**/thing.file, **/thing.file should be this and child directories. 

I'll double check though!

Reply:
(double checked - it does look in the current dir)